lib/models/notification_event.ts

```diff
@@ -13,8 +12,7 @@ import { CreatedMessageEventSenderMetadata } from "./created_message_sender_meta
  * have been configured for that notification.
  */
 export const NotificationEvent = t.interface({
-  messageContent: MessageContent,
-  messageId: NonEmptyString,
+  message: t.intersection([NewMessageWithContent, NotExpiredMessage]),
```

Comment:
notification event must carry a NotExpiredMessage

Reply:
always better avoid naming things in a negative way (can cause confusion in double negation checks), perhaps  `ActiveMessage` ?

Reply:
renamed